### PR TITLE
feat(exchanges): disable edition when it has participants

### LINKED ISSUE #1329: feat(exchanges): disable edition when it has participants
When an exchange has at less one participant 
- [ ] should disable the edition button on card
- [ ] should add a title telling the user that exchange with participent are not editable
- [ ] the fields should be read only on the exchange edition form (at `/@~/exchanges/xxx/edit` )

### PR DESCRIPTION
**Pull request by JNSPK**
fixes #1329

### CODE REVIEW

File: apps/www/widgets/exchanges/card/index.tsx [hunk 1]
```diff
@@ -78,6 +79,22 @@ function Idle() {
   const exchange = useExchange();
   const { is_yours } = useExchangeMeta();
 
+  console.log(exchange);
```
Comment (douglasduteil) [inline apps/www/widgets/exchanges/card/index.tsx:82]: remove console log

File: apps/www/widgets/exchanges/card/index.tsx [hunk 2]
```diff
@@ -78,6 +79,22 @@ function Idle() {
...
+
+  const [showInfoBox, setShowInfoBox] = useState(false);
+
+  const alreadyPopulated = exchange.deals.length > 0;
+
+  const handleClick = (e: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => {
```
Comment (douglasduteil) [inline apps/www/widgets/exchanges/card/index.tsx:88]: Directly import the type `MouseEvent` without React namespace

File: packages/@1/modules/exchange/ui/src/form/new_exchange.tsx [hunk 3]
```diff
@@ -35,15 +36,18 @@ export function Exchange_CreateForm<
   isSubmitting,
   setFieldValue,
   values,
+  alreadyPopulated,
 }: Exchange_CreateForm_Props<TValues>) {
+  const isDisabled = isSubmitting || alreadyPopulated;
```
Comment (douglasduteil) [inline packages/@1/modules/exchange/ui/src/form/new_exchange.tsx:41]: 👍

**Review by douglasduteil — CHANGES_REQUESTED** [review]

File: apps/www/widgets/exchanges/card/index.tsx (hunk 1 above)
Comment (JNSPK) [reply apps/www/widgets/exchanges/card/index.tsx:82]: fix: oups

**Review by douglasduteil — APPROVED** [review]

**Pull request merged**